test/SparqlParserTest.cpp

```diff
@@ -1480,11 +1482,26 @@ TEST(ParserTest, parseWithDatasets) {
                       m::SelectQuery(m::VariablesSelect({"?x"}, false, false),
                                      filterGraphPattern)),
           datasets));
-}
-
-TEST(ParserTest, multipleUpdatesAreForbidden) {
-  AD_EXPECT_THROW_WITH_MESSAGE(
-      SparqlParser::parseQuery(
-          "INSERT DATA { <a> <b> <c> }; DELETE DATA { <d> <e> <f> }"),
-      testing::HasSubstr("Multiple Updates in one request are not supported."));
+  auto deleteWhereOp =
+      m::GraphUpdate({SparqlTripleSimpleWithGraph{Var("?s"), Var("?p"),
+                                                  Var("?o"), std::monostate{}}},
+                     {}, std::nullopt);
+  auto deleteWherePattern =
+      m::GraphPattern(m::Triples({{Var("?s"), "?p", Var("?o")}}));
```

Comment:
This line currently makes the unit test fail, as the `"?p"` has to be wrapped in `Var()` since a very recent refactoring.